Justfile

```diff
@@ -8,9 +8,12 @@ setup:
     git submodule update --init --recursive
   fi
   if [[ "$(cargo 2>&1)" == *"rustup could not choose a version of cargo to run"* ]]; then
-    rustup default 1.74.0
+    rustup default 1.76.0 # TODO undo this
```

Comment:
@diehuxx I know this is no bueno but thinking of leaving it for now. I had to do it because of the `wasm-pack` version we're using... though we could look into moving to a lower `wasm-pack` I'm not sure what the consequences would be. Lmk your thoughts.

Reply:
That's fine. We CI job that checks if the web5 crate has MSRV 1.74. The CI job checks `crates/web5/Cargo.toml` to get `rust-version = 1.74`, so as long as we don't change that, we're fine.

The only drawback of using a higher rust version during development is that we may accidentally use features in `crates/web5` that aren't supported in version 1.74. If that happens, we'll just notice them in the CI job failure and fix afterward.